src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Validation.cs

```diff
@@ -122,18 +124,26 @@ internal static unsafe partial class Utf8Utility
 
                         do
                         {
-                            if (Sse2.IsSupported)
+                            if (AdvSimd.Arm64.IsSupported)
+                            {
+                                mask = AdvSimd.Arm64.MaxAcross(AdvSimd.LoadVector128(pInputBuffer)).ToScalar();
```

Comment:
You'll have to copy the function declarations(and throw a PNSE) for each of these methods. Similar to what I'm doing in https://github.com/dotnet/runtime/pull/38597

Reply:
Thanks yes, the CI failed.